oephys2nwb/export2nwb.py

```diff
@@ -761,13 +761,20 @@ def export2nwb(data_dir : str,
                                resolution=1/eInfo.sampleRate,
                                description="TTL pulse values")
                 nwbfile.add_acquisition(ttlData)
-                ttlChan = TTLs(name="TTL_ChannelStates",
+                ttlChan = TTLs(name="TTL_Channels",
+                               data=evtPd.channel.to_numpy(),
```

Comment:
Makes perfect sense. Do you think it makes sense to drop the pulse values here, @KatharineShapcott ? Are they relevant?

Reply:
Okay, got it - thank you!